notebooks/sippy-analysis.ipynb

```diff
@@ -0,0 +1,2411 @@
+{
```

Comment:
<p>Can you move this notebook to <code>notebooks/data-sources/Sippy/stage/</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/53/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>